packages/user/Symbol/service/src/lib.rs

```diff
@@ -325,7 +335,12 @@ pub mod service {
 
     #[action]
     fn create(symbol: AccountNumber) {
-        Symbol::add(symbol);
+        Symbol::add(symbol, get_sender());
+    }
+
+    #[action]
+    fn admin_create(symbol: AccountNumber, recipient: AccountNumber) {
```

Comment:
I don't understand this... 
1. you don't need an `admin_create` action if the admin is detected as `sender == get_service()`.
2. You don't need a recipient parameter, the owner of tokens created by the admins is the symbol service.

Reply:
Rename to `create_for` and make it callable by the general public.